Sources/LocalReceiptParsing/BasicTypes/InAppPurchase.swift

```diff
@@ -45,6 +45,16 @@ extension AppleReceipt.InAppPurchase {
         return expiration > Date()
     }
 
+    var purchaseDateEqualsExpiration: Bool {
+        guard self.productType?.isSubscription == true else { return false }
+        guard let expiration = self.expiresDate else { return false }
+
+        return abs(self.purchaseDate.timeIntervalSince(expiration)) <= Self.purchaseAndExpirationEqualThreshold
+    }
+
+    /// Seconds between purchase and expiration to consider both equal.
+    private static let purchaseAndExpirationEqualThreshold: TimeInterval = 5
```

Comment:
I thought when this happened, the dates would be exactly the same (based on the example). Is there any risk of false positives here when time rate is accelerated in SKTest, with valid expiration dates?

Reply:
oh I see, makes sense. Yeah, documenting the decision there would be helpful

Reply:
That's done :)